flow-client/src/main/frontend/vaadin-dev-tools.ts

```diff
@@ -1119,8 +1119,11 @@ export class VaadinDevTools extends LitElement {
     );
 
     if ((window as any).Vaadin) {
+      const existing = (window as any).Vaadin.devTools;
       (window as any).Vaadin.devTools = this;
+      Object.assign((window as any).Vaadin.devTools, existing);
```

Comment:
```suggestion
      (window as any).Vaadin.devTools = Object.assign(this, (window as any).Vaadin.devTools);
```

And probably `window as any` is not required inside the `if` branch. I guess TypeScript should infer that?

Reply:
Fixed. TypeScript does not infer that